### PR TITLE
Takehome1

### PR DESCRIPTION
**Pull request by connielion**
@ryekerjh ¯\\__(ツ)__/¯ document.write("I did it?!");

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var logThis = "I'd like to learn the process of making an app, learn another language, and get another step closer to become a front-end developer etc. :D'";
+
+var myValue = confirm(logThis);
+
+console.log(logThis);
```
Comment (ryekerjh) [inline js/script.js]: You don't want to log out the string, you want to log out the value that `confirm(logThis)` returns.

Comment (connielion) [reply js/script.js]: Oh, the “true” or “false”?

Comment (ryekerjh) [reply js/script.js]: Right.

**Comment by connielion** [comment]
@ryekerjh I think I got it for real this time?

**Comment by ryekerjh** [comment]
Nice! You are approved!

**Pull request merged**